client/src/views/landing.vue

```diff
@@ -74,7 +74,7 @@ export default class Landing extends Vue {
 #landing-container
   margin: 0
   padding: 0
-  min-width: 1380px !important
+  min-width: 1080px !important
```

Comment:
Will this be responsive on a smaller screen?

Reply:
Slightly, working towards that. This was the point when the main container started to break, for me at least.